css/style.css

```diff
@@ -0,0 +1,220 @@
+/* 
+Spacing system (px)
+2 / 4 / 8 / 12 / 16 / 24 / 32 / 48 / 64 / 80 / 96 / 128
+
+Font sizes (px)
+10 / 12 / 14 / 16 / 18 / 20 / 24 / 30 / 36 / 44 / 52 / 62 / 74 / 86 / 98
+*/
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+.wrapper {
+  width: 160rem;
+  margin: 0 auto;
+}
+
+/******************************/
+/* HEADER */
+/******************************/
+
+.header {
+  position: absolute;
+  display: flex;
+  width: 160rem;
+  height: 12rem;
+  justify-content: center;
+  align-items: center;
+  padding-left: 18rem;
+  padding-right: 18rem;
+  background: linear-gradient(to bottom, black -100%, transparent 90%);
+  gap: 0.8rem;
+}
+
+.logo-menu-bar {
+  display: flex;
+  justify-content: space-between;
+  gap: 22.8rem;
+  align-items: center;
+}
+
+.logo {
+  font-family: "Yeseva One", sans-serif;
+  text-decoration: none;
+  color: #fff;
+  text-transform: uppercase;
+  font-size: 3.6rem;
+  opacity: 0.9;
```

Comment:
Why everywhere `opacity` in percentage but not it's not?

Reply:
Fixed. Did not notice the loss of the sign "%".